src/VisualStudio/Core/Def/Implementation/Remote/SolutionChecksumUpdater.cs

```diff
@@ -102,17 +111,12 @@ private void OnWorkspaceChanged(object sender, WorkspaceChangeEventArgs e)
         private void EnqueueChecksumUpdate()
         {
             // event will raised sequencially. no concurrency on this handler
```

Comment:
```suggestion
            // event will raised sequentially. no concurrency on this handler
```

Reply:
I don't currently have my spell checking extension installed. I'm good leaving this for a separate change focused on spelling and grammar.